### PR TITLE
Graduate melgu

### PR DESCRIPTION
**Pull request by melgu**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/melgu/melgu.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Melvin Gundlach
+institution: Universität Hamburg
```
Comment (Mohammed-Shoaib) [inline _data/melgu/melgu.md:3]: Change the _ä_ to _a_ as it is a non-ascii character.

**Comment by melgu** [comment]
It's sad that all of Germany's universities ("Universitäten") are excluded, but so be it. I'll change it to "Uni".

**Comment by Mohammed-Shoaib** [comment]
@melgu Thank you for changing it! That's really sad, you could still use _University of Hamburg_ if you'd like instead of _Uni Hamburg_!

**Comment by melgu** [comment]
_Uni Hamburg_ is fine. Thanks for your efforts 👍

**Comment by melgu** [comment]
Why did the pictures need to have that resolution, if the stream is in 720p and even the text is blurry?

**Pull request closed without merging**